packages/flow/src/css/component/FlowConnection.module.scss

```
@@ -0,0 +1,67 @@
+    stroke-linejoin: round;
+    cursor: pointer;
+    pointer-events: stroke;
+}
+
+.flowConnectionLine {
+    fill: none;
+    stroke: var(--connection-color);
+    stroke-width: 2px;
+    stroke-linecap: round;
+    stroke-linejoin: round;
+    // A zero gap renders solid, but expresses the solid state as a dash pattern so
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)